src/redux/wallets.ts

```diff
@@ -56,7 +55,7 @@ interface WalletsState {
   /**
    * The current loading state of the wallet.
    */
-  isWalletLoading: WalletLoadingState | null;
+  isWalletLoading: any;
```

Comment:
curious what this is being used for still

Reply:
Just wanted to keep file changes down, but tbh might only be 1-3 more files.

Reply:
Eh I don't wanna axe all that old portal stuff, I don't fully grok it yet